File: TAP_U3ProyectoFinalDali/src/main/java/com/mycompany/domain/Usuario.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.domain;

/**
 *
 * @author charl
 */
public class Usuario {
    private int idUsuario;
    private String username;
    private String password;
    private String tipo_usuario;

    public Usuario() {
    }

    public Usuario(String username, String password, String tipo_usuario) {
        this.username = username;
        this.password = password;
        this.tipo_usuario = tipo_usuario;
    }

    public Usuario(int idUsuario, String username, String password, String tipo_usuario) {
        this.idUsuario = idUsuario;
        this.username = username;
        this.password = password;
        this.tipo_usuario = tipo_usuario;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getTipo_usuario() {
        return tipo_usuario;
    }

    public void setTipo_usuario(String tipo_usuario) {
        this.tipo_usuario = tipo_usuario;
    }

    @Override
    public String toString() {
        return "Usuario{" + "idUsuario=" + idUsuario + ", username=" + username + ", password=" + password + ", tipo_usuario=" + tipo_usuario + '}';
    }
    
    
}

File: WS_U3_AppDiplomas/Usuario.php
<?php

class Usuario {
    private $idUsuario;
    private $username;
    private $password;
    private $tipo_usuario;

    public function _construct() {
    }

    public function _construct2($username, $password, $tipo_usuario) {
        $this->username = $username;
        $this->password = $password;
        $this->tipo_usuario = $tipo_usuario;
    }

    public function _construct3($idUsuario, $username, $password, $tipo_usuario) {
        $this->idUsuario = $idUsuario;
        $this->username = $username;
        $this->password = $password;
        $this->tipo_usuario = $tipo_usuario;
    }

    public function getIdUsuario() {
        return $this->idUsuario;
    }

    public function setIdUsuario($idUsuario) {
        $this->idUsuario = $idUsuario;
    }

    public function getUsername() {
        return $this->username;
    }

    public function setUsername($username) {
        $this->username = $username;
    }

    public function getPassword() {
        return $this->password;
    }

    public function setPassword($password) {
        $this->password = $password;
    }

    public function getTipo_usuario() {
        return $this->tipo_usuario;
    }

    public function setTipo_usuario($tipo_usuario) {
        $this->tipo_usuario = $tipo_usuario;
    }

    public function __toString() {
        return "Usuario{" . "idUsuario=" . $this->idUsuario . ", username=" . $this->username . ", password=" . $this->password . ", tipo_usuario=" . $this->tipo_usuario . '}';
    }
}


?>